app.js

```diff
@@ -101,12 +157,14 @@ function surveyItemContainerClick(e){
       item.numberOfClicks++;
     }
   });
-  if(itemsChosen < 25){
+  if(itemsChosen < 24){
     itemsChosen++;
     displayRandomItems();
-    console.log(itemObjects);
+    writeToStorage();
```

Comment:
If you have the same code in both/all branches of an `if`/`else`, consider the code can be moved before/after.